java/org/contikios/mrm/ChannelModel.java

```diff
@@ -1448,15 +1448,16 @@ private double[] getTransmissionData(TxPair txPair, TransmissionData dataType) {
     }
 
     // - Calculate total path loss (using simple Rician) -
-    double[] pathModdedLengths = new double[allPaths.size()];
+    var numPaths = allPaths.size();
```

Comment:
`allPaths.size()` is used multiple times before this point. Why not put this where `allPaths.size()` is first called?